cpp/tests/quantiles/tdigest_utilities.cu

```diff
@@ -51,23 +51,44 @@ void tdigest_sample_compare(cudf::tdigest::tdigest_column_view const& tdv,
   auto sampled_result_weight = cudf::make_fixed_width_column(
     data_type{type_id::FLOAT64}, h_expected.size(), mask_state::UNALLOCATED);
 
-  rmm::device_vector<expected_value> expected(h_expected.begin(), h_expected.end());
```

Comment:
just curious: where is expected_value defined? I guess it's a thrust::tuple?

Reply:
https://github.com/rapidsai/cudf/blob/5a4afecad5ff2769fc7f32565158e1321c358b71/cpp/include/cudf_test/tdigest_utilities.cuh#L44